recipes/readabilipy/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license file is already in the source distribution, so do not need to include it here.